.github/stale.yml

```diff
@@ -0,0 +1,11 @@
+daysUntilStale: 60
```

Comment:
```suggestion
daysUntilStale: 14
```

Reply:
Yeah, I was worried that this might be too quick but let's try it.